hack/machine_types/machine_types.go

```diff
@@ -193,6 +200,14 @@ func run() error {
 					machine.ECU = stringToFloat32(attributes["ecu"])
 				}
 
+				// AWS VPC CNI plugin-specific maximum pod calculation based on:
+				// https://github.com/aws/amazon-vpc-cni-k8s/blob/f52ad45/README.md
+				enisPerInstance, enisOK := awsutils.InstanceENIsAvailable[attributes["instanceType"]]
```

Comment:
If the license is compatible, could we just copy these maps.  The dependency seems non-trivial, and I'd also like to make the file we're generating here canonical.  We just haven't figured out where to put it yet!  Maybe we need a kubernetes-sigs project `kubernetes-sigs/aws-commons`

Reply:
And it looks like the license is compatible - Apache 2.  I think that makes it OK to just copy the file, for example.

Reply:
I'll import the maps. 👍